### DISCUSSION TITLE
Hot to set Podcast link/page?

### DISCUSSION THREAD

**Discussion by sithumonline** [sliit-foss/sliitfoss#141] [Q&A] [answered]
Link https://anchor.fm/ directly to navbar.

![Screenshot_2021-01-29_10-03-06](https://user-images.githubusercontent.com/24352487/106232567-559daa80-621a-11eb-965e-fdbb9e8f6893.png)

![Screenshot_2021-01-29_10-08-39](https://user-images.githubusercontent.com/24352487/106232581-5afaf500-621a-11eb-8507-b3e959f378e2.png)

**Comment by chamodshehanka** [comment, marked as answer]
Since we're planning to have more platforms this won't be a good idea
